cves/kernel/CVE-2016-5728.yml

```diff
@@ -55,7 +55,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: A race condition in one of the functions in a certain driver that allows 
+  users to obtain senstive information from memory or cause memory corruption or a system 
+  crash. If the user modifies the header of a file, the function might incorrectly fetch 
```

Comment:
Were any examples given of how a user could modify the header file could be modified to cause this vulnerability? The way it is described here sounds to me like a developer would have to update the header file, which won't affect code until the program is recompiled.